specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,4002 @@
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "GlobalReachConnection": {
+      "description": "A global reach connection resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of a global reach connection resource",
+          "$ref": "#/definitions/GlobalReachConnectionProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "GlobalReachConnectionProperties": {
+      "description": "The properties of a global reach connection",
+      "properties": {
+        "provisioningState": {
+          "description": "The state of the  ExpressRoute Circuit Authorization provisioning",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "GlobalReachConnectionProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "addressPrefix": {
+          "description": "The network used for global reach carved out from the original network block provided for the private cloud",
+          "type": "string",
+          "readOnly": true
+        },
+        "authorizationKey": {
+          "description": "Authorization key from the peer express route used for the global reach connection",
+          "type": "string"
+        },
+        "circuitConnectionStatus": {
+          "description": "The connection status of the global reach connection",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Connected",
+            "Connecting",
+            "Disconnected"
+          ],
+          "x-ms-enum": {
+            "name": "GlobalReachConnectionStatus",
+            "modelAsString": true
+          }
+        },
+        "peerExpressRouteCircuit": {
+          "type": "string",
+          "description": "Identifier of the ExpressRoute Circuit to peer with in the global reach connection"
+        }
+      }
+    },
+    "WorkloadNetworkSegmentSubnet": {
+      "description": "Subnet configuration for segment",
+      "properties": {
+        "dhcpRanges": {
+          "description": "DHCP Range assigned for subnet.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "gatewayAddress": {
+          "description": "Gateway address.",
+          "type": "string"
+        }
+      }
+    },
+    "WorkloadNetworkSegmentPortVif": {
+      "description": "Ports and any VIF attached to segment.",
+      "properties": {
+        "portName": {
+          "description": "Name of port or VIF attached to segment.",
+          "type": "string"
+        }
+      }
+    },
+    "WorkloadNetworkSegmentProperties": {
```

Comment:
the PUT for this resource is marked as long running.  The properties should contain "provisioningState" if that is true

Reply:
There seems to be at least a few other swagger specs with proxy resources that have puts and don't include the the "provisioningState" field and at least one that I looked at (SQL ServerAzureADAdministrator) is also marked as a long running operation. What exactly determines if you need this field or not? Here are some of the examples I was looking at:

Linked Services:
def -> https://github.com/Azure/azure-rest-api-specs/blob/64541c6af93ae1adbbb4875ffea97e310963687e/specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/LinkedServices.json#L37
ex -> https://github.com/Azure/azure-rest-api-specs/blob/64541c6af93ae1adbbb4875ffea97e310963687e/specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/examples/LinkedServicesCreate.json

SQL TransparentDataEncryption:
def -> https://github.com/Azure/azure-rest-api-specs/blob/2f7b225e23442d0ade36bf15d4ac15730b683c9c/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/sql.core.json#L207
ex -> https://github.com/Azure/azure-rest-api-specs/blob/2f7b225e23442d0ade36bf15d4ac15730b683c9c/specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/examples/DatabaseTransparentDataEncryptionCreateOrUpdate.json

SQL ServerAzureADAdministrator:
def -> https://github.com/Azure/azure-rest-api-specs/blob/98c74125ced2106024ec5a837d882c56e46765f5/specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ServerAzureADAdministrators.json#L71
ex -> https://github.com/Azure/azure-rest-api-specs/blob/98c74125ced2106024ec5a837d882c56e46765f5/specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/examples/AdministratorCreateOrUpdate.json

AZ Portal tenantConfiguration:
def -> https://github.com/Azure/azure-rest-api-specs/blob/ff5a5ea9c0eafa289525fe2607266e358ed00b59/specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/tenantConfiguration.json#L97
ex -> https://github.com/Azure/azure-rest-api-specs/blob/ff5a5ea9c0eafa289525fe2607266e358ed00b59/specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/examples/TenantConfiguration/CreateOrUpdateTenantConfiguration.json